ui/components/configuratorComponents/CodeEditor.js

```diff
@@ -19,10 +20,18 @@ const useStyles = makeStyles(theme => ({
       minWidth : "calc(50% - 175px)",
       maxWidth : "calc(50% - 175px)",
     },
+  },
+  icon : {
+    position : "absolute",
+    right : "24px",
+    bottom : "30px",
+    color : "#fff",
+    zIndex : 999,
```

Comment:
Is z-index 1 already taken by a competing element? If not, please put this at z-index 1

Reply:
Yes, 1 is code-editor.
For the level aspect: Putting 11 Now :smiley: